keel/quotations/admin.py

```diff
@@ -16,7 +28,38 @@ class QuotationAdmin(admin.ModelAdmin):
 
 
 class QuotationMilestioneAdmin(CustomBaseModelAdmin):
-    pass
+    
+    def save_model(self, request, obj, form, change):
+        id = str(obj.pk)
+        customer = obj.quotation.user
+        
+        try:
+            case = Case.objects.get(user=customer)
+        except Exception as e:
```

Comment:
Try to use specific exception as much as possible than generic ones. You could use ObjectDoesNotExists exception  handler here.